Server/src/controllers/getDataController.ts

```diff
@@ -0,0 +1,12 @@
+import { retrieveData } from '../services/getDataService';
+
+
+export const createRequestForData = async (req, res) => {
```

Comment:
Still needs to be converted to class

Reply:
Good catch, rectified this in commit 3c228c5